lib/datadog/tracing/contrib/redis/instrumentation.rb

```diff
@@ -25,6 +25,20 @@ def call(*args, &block)
                 span.resource = get_command(args, show_command_args)
                 ### BRAZE MODIFICATION
                 span.set_metric Contrib::Redis::Ext::METRIC_RAW_COMMAND_LEN, args.to_s.length
+
+                if Thread.current[:redis_operation_filepath].present?
+                  span.set_tag(
+                    Contrib::Redis::Ext::METRIC_FILEPATH,
+                    Thread.current[:redis_operation_filepath]
```

Comment:
What do you think about defining `:redis_operation_filepath` and `:redis_operation_codeowner` as constants in redis/ext.rb (and perhaps referenced in `platform` as well)?